.circleci/config.yml

```diff
@@ -153,6 +76,7 @@ workflows:
             branches:
               ignore:
                 - /rel\/.*/
+                - master
```

Comment:
Don't we still want the tests running upon each merge to master?

Reply:
_Maybe_...we do catch it in the nightlies. However, I can pull this out if you think it adds value. It just seems weird to re-run it every time we merge a PR.